index.bs

```diff
@@ -1432,8 +1429,131 @@ following additional fields:
+    receive, or if they are received automatically by the receiver,
+    give the user some information about what th e media stream is.
+
+: audio
+:: A list of audio encodings offered.  An audio encoding is a series
+    of encoded audio frames.  Encodings define properties needed by
+    the receiver to know how to decode the encoding, such as codec and
+    sample rate.  They can differ by codec and related properties,
+    but should be different encodings of the same audio.
+
+: video
+:: A list of video encodings offered.  A video encoding is a series of
+    encoded video frames.  Encodings define properties needed by the
+    receiver to know how to decode the encoding, such as codec and
+    default duration.  They can differ by codec and potentially other
+    properties, but should be different encodings of the same video.
+
+
+Each audio encoding offered defines the following properties:
+
+: encoding-id
+:: Identifies the audio encoding being offered.  Must be unique within
+    the media stream.  Can be the receiver to request an encoding.
+
+: codec
+:: The name of the codec used by the encoding.
+
+: time-scale
+:: The time scale used by all audio frames.  This allows senders to
+    make audio-frame messages smaller by not including the time scale
+    in each one.
+
+: default-duration:
+:: The duration of an audio frame .  This allows senders to make
+    audio-frame messagse smaller by not including the duration for
+    audio-frame messages that have the default duration.
+
+Each video encoding offered defines the following properties:
+
+: encoding-id
+:: Identifies the video encoding being offered.  Must be unique within
+    the media stream.  Can be the receiver to request an encoding.
+
+: codec
+:: The name of the codec used by the encoding.
+
+: time-scale
+:: The time scale used by all video frames.  This allows senders to
+    make video-frame messages smaller by not including the time scale
+    in each one.
+
+: default-duration:
+:: The default duration of a video frame .  This allows senders to make
+    video-frame messagse smaller by not including the duration for
+    video-frame messages that have the default duration.
+
+: default-rotation:
+:: The default rotation of a video frame .  This allows senders to make
+    video-frame messagse smaller by not including the rotation for
+    video-frame messages that have the default rotation.
+
+After receiving a streaming-session-start-request message, a receiver
+should send back a streaming-session-start-response message with the
+following properties:
+
+: stats-interval
+:: Indicates the frequency the sender should send stats messages to
+    the receiver.
+
+: stream-requests
+:: Indicates which media streams the receiver would like to receiver
+    from the sender.
+
+Each stream request contains the following properties:
+
+: media-stream-id
+:: The ID of the stream reqeusted.
+
+: audio (optional)
+:: The requested audio encoding, by encoding ID
```

Comment:
Is the receiver allowed to ask for subsampled audio, because of hardware limitations?  For example, 16 kHz instead of an offered encoding of 48 Khz.

Or, maybe the sender can offer multiple audio encodings at reasonable subsampling rates, if it is willing to subsample.

Reply:
The later, but I believe Opus is always 48khz.   Downsampling before playout on the receiver isn't a big deal.

Reply:
SGTM